t.cs

```diff
@@ -0,0 +1,40 @@
+public class Test
+{
+    public bool Enabled
+    {
+        get
+        {
+
+            return true;
+        }
+    }
+
+    public bool T(bool c)
+    {
+        do
+        {
+            Console.WriteLine("Loop forever");
+        }
+
+        while (true);
+
+        if (c)
+        {
+            T(false);
+        }
```

Comment:
Closing brace should be followed by blank line.

```suggestion
        }

```